knpu/episode1/test-code-reuse.md

```diff
@@ -21,6 +21,7 @@ And give `Client` a proper `use` statement:
 
 [[[ code('9263e01200') ]]]
 
+In case you are using Guzzle 6, you would need to pass the base to `base_uri` instead of `base_url`.
```

Comment:
Could you change it to the:
```
***TIP
In case you are using Guzzle 6, you would need to use the `base_uri` key instead of `base_url` to configure Guzzle client properly.
***
```

Nice styles will be applied to the tip if we wrap it with this code.